ibm_catalog.json

```diff
@@ -21,20 +21,21 @@
         "ai"
       ],
       "short_description": "Creates and configures an instance of IBM watsonx.data.",
-      "long_description": "This architecture supports creating and configuring an instance of watsonx.data.",
+      "long_description": "This architecture supports creating and configuring an instance of watsonx.data. For more details on watsonx data, [see here](https://cloud.ibm.com/docs/watsonxdata/index.html).",
```

Comment:
Good to have one or two lines for watsonx.data. You can use its documentation. Idea here is  to show the value of your DA then only it will be used. Also, how it is different than the catalog's watsonx.data tile.